app/src/main/java/io/homeassistant/companion/android/widgets/camera/CameraWidget.kt

```diff
@@ -129,18 +67,18 @@ class CameraWidget : AppWidgetProvider() {
                 val baseUrl = serverManager.getServer(widget.serverId)?.connection?.getUrl().toString().removeSuffix("/")
                 val url = "$baseUrl$entityPictureUrl"
                 if (entityPictureUrl == null) {
-                    setImageViewResource(
-                        R.id.widgetCameraImage,
-                        R.drawable.app_icon_round
-                    )
-                    setViewVisibility(
-                        R.id.widgetCameraPlaceholder,
-                        View.VISIBLE
-                    )
-                    setViewVisibility(
-                        R.id.widgetCameraImage,
-                        View.GONE
-                    )
+//                    setImageViewResource(
+//                        R.id.widgetCameraImage,
```

Comment:
We should not leave commented code, this looks like it should be used though right?